### PR TITLE
feature: Base Login page + removal of print to console from Registration

### PR DESCRIPTION
**Pull request by mattydono**
- Basic Login page added currently only tracking changes in username + password fields.

- Print button + console.log function removed from Registration as all working as expected.

### CODE REVIEW

File: src/Login/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  public render() {
+    return (
+      <div>
+        <input
+          placeholder='Email'
+          value={this.state.email}
+          onChange={this.onEmailChange}
+        />
+        <input
+          placeholder='Password'
```
Comment (Sharakai) [inline src/Login/Login.tsx:23]: This should have a `type="password"`, else it isn't obfuscated and hidden from prying eyes.

File: src/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export class Login extends Component {
+  public state: State = {
+    email: '',
+    password: '',
+  };
+
+  public render() {
+    return (
+      <div>
```
Comment (Sharakai) [inline src/Login/Login.tsx:16]: No styling of any sort?

File: src/Login/Login.tsx (hunk 1 above)
Comment (mattydono) [reply src/Login/Login.tsx:23]: Got it!

**Comment by Sharakai** [comment]
Don't start commit messages with a capital letter, when possible.
I'll include `commitlint` at some point, to force you to use good commit messages, but just bear it in mind for the future.

File: src/Login/Login.tsx (hunk 2 above)
Comment (mattydono) [reply src/Login/Login.tsx:16]: I was going to get all of the functionality in and working before starting on any styling.

Realistically I'll style once I have react router in place and all my pages nicely separated.

**Comment by mattydono** [comment]
No problem, I'll make sure not to continue with this going forward.

**Review by Sharakai — APPROVED** [review]

**Pull request merged**